pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -94,3 +96,49 @@ func validateNamespace(namespace string, client Client, fieldPath *field.Path) f
 
 	return allErrs
 }
+
+func validateHcoFeatureGates(client Client, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	hcUnstructured, err := client.GetHyperConverged(context.Background(), "kubevirt-hyperconverged", "openshift-cnv")
+	if err != nil {
+		allErrs = append(
+			allErrs,
+			field.NotFound(
+				fieldPath,
+				fmt.Sprintf("failed to get resource openshift-cnv/kubevirt-hyperconverged, with error: %v", err),
+			),
+		)
+
+		return allErrs
+	}
+
+	bytes, err := hcUnstructured.MarshalJSON()
+	if err != nil {
+		allErrs = append(
+			allErrs,
+			field.InternalError(
+				fieldPath,
+				err,
+			),
+		)
+
+		return allErrs
+	}
+
+	enabled, ok := gojsonq.New().FromString(string(bytes)).Find("spec.featureGates.HotplugVolumes").(bool)
```

Comment:
Rather than introducing a new module, use the methods in the the https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/unstructured package. This has the added benefit of also not having to re-marshal the data, too.

```suggestion
	enabled, exists, err := unstructured.NestedBool(hcUnstructured, "spec", "featureGates", "HotplugVolumes")
```

Reply:
Thanks. That's exactly what I was looking for.